.gitignore

```diff
@@ -1,3 +1,5 @@
+.gradle/
```

Comment:
Why do we need to ignore a gradle directory in a node project?

Reply:
Good point. We don't use gradle in web-ui so this isn't needed. @adeel718 you can just delete the .gradle directory on your local machine.